File: app/src/main/java/com/shuoye/video/pojo/TimeLine.java
package com.shuoye.video.pojo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

/**
 * 新番时间表资源
 *
 * @author shuoye
 * @program video
 * @ClassName TimeLine
 * @create 2021-10-16 18:36
 **/
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Accessors(chain = true)
public class TimeLine {
	private Integer id;
	private String name;
	private String episodes;
	private String cover;
}

File: app/src/main/java/com/shuoye/video/database/pojo/TimeLine.kt
package com.shuoye.video.database.pojo

import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * 新番时间表实体类
 * @program Video
 * @ClassName TimeLine
 * @author shuoye
 * @create 2021-10-21 20:48
 **/
@Entity(tableName = "time_line")
data class TimeLine(
    @PrimaryKey
    val id: Int,
    val isNew: Boolean,
    val time: String,
    val name: String,
    val nameForNew: String,
    val wd: Int

)
